README.md

```diff
@@ -385,6 +456,117 @@ Installation Instructions
   ```
 
 
+JSON inventory method
+----------------------
+
+ There is also created Ansible playbook sample for deploying IBM Spectrum Scale (GPFS) cluster using json inventory.
+ 
+ **playbook_json.ces.yml** --> **set_json_variables.yml** --> **vars/scale_clusterdefinition.json**
+ 
+- **Ansible Playbook:** 
+    - **playbook_json.ces.yml**
+         -  Spectrum Scale Core, ZIMON, GUI, protocol (NFS, SMB), CallHome and Scale file audit logging. 
+ 
+ 
+- **set_json_variables.yml**
+
+    - The Playbook set variables from `set_json_variables.yml` that is read from `vars/scale_clusterdefinition.json`
+
+- **vars/scale_clusterdefinition.json**
+ 
+   - This file can be adjusted to your environment or created.
+   - Example scale_clusterdefinition.json` is separated into:
+   
+       - `scale_cluster`: 
+       - `node_details`: Variables that set's variables to each node (*host_vars*)
+       - `scale_storage`:
+       - `scale_callhome_params`:
+       - `scale_protocols`:
+       
+ 
+  ```json
+    {
+      "scale_cluster": {
+            "scale_cluster_name": "gpfs1.local",
+            "scale_cluster_profile_name": "gpfsprotocoldefaults"
+      },
+      "node_details": [
+        {
+          "fqdn" : host-vm1,
+          "is_protocol_node" :false,
+          "is_nsd_server" : false,
+          "is_quorum_node" : false,
+          "is_manager_node" : false,
+          "is_gui_server" : false,
+          "is_callhome_node" : false,
```

Comment:
Are these variable names correct? I thought that it would be `scale_cluster_quorum` and `scale_cluster_manager`, as explained above... Or was this changed?